.cirrus.yml

```
@@ -2,7 +2,8 @@ only_if: changesInclude('.cirrus.yml') || ($CIRRUS_BRANCH == "main" && !changesI
 
 run_tests: &RUN_TESTS
   install_cibuildwheel_script:
-    - python -m pip install -e ".[dev]" pytest-custom-exit-code
+    - python -m pip install dependency-groups
+    - python -m dependency_groups dev | xargs python -m pip install -e. pytest-custom-exit-code
```

```suggestion
    - python -m dependency_groups test | xargs python -m pip install -e.
```
We should probably align everything to use `test` rather than `dev`.
Is `pytest-custom-exit-code` really needed on some CI ? The usage section on PyPI seems to indicate that it requires to call `pytest` with specific args to get it to do something but we're not (and nothing in the conf either unless I missed something ?).